src/schema_generators/generators.cljc

```diff
@@ -106,25 +110,30 @@
     s/Num (generators/one-of [generators/large-integer generators/double])
     s/Int (generators/one-of
            [generators/large-integer
-            (generators/fmap unchecked-int generators/large-integer)
-            (generators/fmap bigint generators/large-integer)])
+            #?@(:clj [(generators/fmap unchecked-int generators/large-integer)
+                      (generators/fmap bigint generators/large-integer)])])
     s/Keyword generators/keyword
-    clojure.lang.Keyword generators/keyword
+    #?(:clj clojure.lang.Keyword
+       :cljs cljs.core/Keyword) generators/keyword
     s/Symbol (generators/fmap (comp symbol name) generators/keyword)
-    Object generators/any
+    #?(:clj Object :cljs js/Object) generators/any
     s/Any generators/any
     s/Uuid generators/uuid
-    s/Inst (generators/fmap (fn [ms] (java.util.Date. ms)) generators/int)}
-   (into {}
-         (for [[f ctor c] [[doubles double-array Double]
-                           [floats float-array Float]
-                           [longs long-array Long]
-                           [ints int-array Integer]
-                           [shorts short-array Short]
-                           [chars char-array Character]
-                           [bytes byte-array Byte]
-                           [booleans boolean-array Boolean]]]
-           [f (generators/fmap ctor (generators/vector (macros/safe-get +primitive-generators+ c)))]))))
+    s/Inst (generators/fmap (fn [ms] (#?(:clj java.util.Date. :cljs js/Date.) ms)) generators/int)}
+   #?(:clj (into {}
```

Comment:
Nope,  should be good.  For reference, the builtin platform-specific schema types are here:

https://github.com/plumatic/schema/blob/master/src/cljx/schema/core.cljx#L167